dotcom-rendering/src/web/components/DynamicFast.tsx

```diff
@@ -434,7 +412,7 @@ export const DynamicFast = ({
 		: groupedTrails.standard.splice(0, standardsDisplayConfig.columns * 3);
 
 	return (
-		<>
+		<Stack space={3}>
```

Comment:
Stack is a neat abstraction from Source (which I didn't know about!) but if we use it here are we then adding an extra div? If we have the `ul` tag add the spacing below itself it feels like we will end up with more semantic html structure

Reply:
I think we should strive for the flattest and most semantic DOM. In this case, we have already deeply nested [`ul`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/ul) which do not really adequately represent a front card.

I’m happy to separate the introduction of `Stack` into its own PR, though.

Reply:
I’ve removed `<Stack>` from this PR for the time being.